Makefile

```diff
@@ -224,9 +224,9 @@ ifeq ($(python_mn_version),3.6)
   check_reqs_packages := pip_check_reqs virtualenv tox pipdeptree build pytest coverage coveralls flake8 pylint twine jupyter notebook
 else
 ifeq ($(python_mn_version),3.7)
-  check_reqs_packages := pip_check_reqs virtualenv tox pipdeptree build pytest coverage coveralls flake8 pylint twine jupyter notebook safety
+  check_reqs_packages := pip_check_reqs virtualenv tox pipdeptree build pytest coverage coveralls flake8 pylint twine jupyter notebook safety towncrier
 else
-  check_reqs_packages := pip_check_reqs virtualenv tox pipdeptree build pytest coverage coveralls flake8 pylint twine jupyter notebook safety sphinx
+  check_reqs_packages := pip_check_reqs virtualenv tox pipdeptree build pytest coverage coveralls flake8 pylint twine jupyter notebook safety sphinx towncrier
```

Comment:
minor: the if-then line and the if-else line are almost identical.
Couldn't you do a:
check_reqs_packages := .... 
before an ifneq that only addds that one entry you need for "not 3.7"?

Reply:
Changed it to:
```
check_reqs_base_packages := pip_check_reqs virtualenv tox pipdeptree build pytest coverage coveralls flake8 pylint twine jupyter notebook
ifeq ($(python_m_version),2)
  check_reqs_packages := $(check_reqs_base_packages)
else ifeq ($(python_mn_version),3.5)
  check_reqs_packages := $(check_reqs_base_packages)
else ifeq ($(python_mn_version),3.6)
  check_reqs_packages := $(check_reqs_base_packages)
else ifeq ($(python_mn_version),3.7)
  check_reqs_packages := $(check_reqs_base_packages) safety towncrier
else
  check_reqs_packages := $(check_reqs_base_packages) safety towncrier sphinx
endif
```